src/components/app.jsx

```diff
@@ -1,9 +1,74 @@
-import React from "react";
-import { ReactComponent as Logo } from "../images/sun.svg";
+import React, { useState, useEffect } from "react";
 import "../App.css";
+import CurrentWeather from "./CurrentWeather";
+import FiveDayForecast from "./FiveDayForecast";
+import Nav from "./Nav";
+import UnitCheckbox from "./UnitCheckbox";
 
 function App() {
-  return <h1>Weather</h1>;
+  const [currentView, setCurrentView] = useState("currentWeather");
+  const [lat, setLat] = useState(null);
+  const [lon, setLon] = useState(null);
+  const [err, setErr] = useState(null);
+  const [unit, setUnit] = useState("imperial");
+
+  const OPEN_WEATHER_APP_ID = "62536724de969bf01c1e02a249c958f9";
+
+  useEffect(() => {
+    // Make a one time call to our browser to get current position
+    window.navigator.geolocation.getCurrentPosition(
```

Comment:
Hi James,

When I first saw this requirement:
 _As a user, I would like to search for the current weather in my area_

I immediately remembered that just recently, I've learned and used the Geolocation API in another app. I thought that it would definitely save users the trouble of having to type their current city into a search bar and save some time. However, the drawback with this approach is that currently, this app is only limited to your current location. I think a good future enhancement would be to add a search bar so that users can see current weather and 5 day forecast of other places.